build_tools/autogen_ltc_backend.py

```diff
@@ -213,18 +214,21 @@ def get_opnames(ops):
 
         # use ripgrep if available as its much faster
         if which("rg") is not None:
-            cmd = ["rg", "-o", "-N", r"aten::[0-9a-zA-Z_\.]+"]
+            cmd = ["rg", "-o", "-N", r"aten::[0-9a-zA-Z_\.]+", str(self.torch_ops_file)]
```

Comment:
can we just replace this with native python code instead of shelling out?

Reply:
Got rid of shell execution by `subprocess`. Use `re` to search instead.